website/docs/r/dashboard.html.markdown

```diff
@@ -117,41 +117,45 @@ The are a bunch of use cases where this layout makes things too verbose and hard
 
 The dashboard is divided into equal-sized charts (defined by `width` and `height`). If a chart does not fit in the same row (because the total width > max allowed by the dashboard), this and the next ones will be place in the next row(s).
 
-```terraform
+```tf
 resource "signalfx_dashboard" "grid_example" {
-    name = "Grid"
-    dashboard_group = "${signalfx_dashboard_group.example.id}"
-    time_range = "-15m"
-
-    grid {
-        chart_ids = ["${concat(signalfx_time_chart.rps.*.id,
-                signalfx_time_chart.50ths.*.id,
-                signalfx_time_chart.99ths.*.id,
-                signalfx_time_chart.idle_workers.*.id,
-                signalfx_time_chart.cpu_idle.*.id)}"]
-        width = 3
-        height = 1
-    }
+  name            = "Grid"
+  dashboard_group = signalfx_dashboard_group.example.id
+  time_range      = "-15m"
+
+  grid {
+    chart_ids = [
+      concat(
+        signalfx_time_chart.rps.*.id,
+        signalfx_time_chart.p50ths.*.id,
```

Comment:
> The first character of an identifier must not be a digit, to avoid ambiguity with literal numbers.

https://www.terraform.io/docs/configuration/syntax.html#identifiers